localbuild/meta.yaml

```diff
@@ -77,7 +77,7 @@ requirements:
     - PyMySQL >=0.9.3
     - validate_email
     - multidict
-    - pint
+    - pint <0.23
```

Comment:
do we have an issue what this new pint does?

Reply:
Yes, there were a bunch of test failures after its release last week. I haven't yet looked into what exactly broke though.

This is one reason why https://github.com/conda-forge/mss-feedstock/issues/162 would be a good idea.

Reply:
there will be soon a 8.3.3 including this and then stable merged, so it is matched with the issue for this.

Reply:
The commit will be dropped by itself when I rebase the next time then.